src/monitoring.tf

```diff
@@ -0,0 +1,122 @@
+module "alarm_channel" {
+  source      = "github.com/massdriver-cloud/terraform-modules//aws-alarm-channel"
+  md_metadata = var.md_metadata
+}
+
+resource "aws_cloudwatch_metric_alarm" "read_capacity" {
```

Comment:
Not a blocker, but we should def figure out how to make a metric query field on alarms